README.md

```diff
@@ -12,17 +23,17 @@ Install the library winregistry using the below command:
 
 ## Configuration
 
-The conf.yaml file should be placed in a directory named as the check (winreg_check.d) under the conf.d folder in the Datadog config hierarchy.
+The conf.yaml file should be placed under the conf.d folder in the Datadog config hierarchy.
 
-### conf.yaml description
+### cutom_win_reg.yaml description
 
 Configuration example
 ```yaml
   - registry_key: ssl3.client
     min_collection_interval: 120
-    key_path: "HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Control\\SecurityProviders\\SCHANNEL\\Protocols\\SSL\ 3.0\\Client"
-    key_name: "Enabled"
-    metric_name: "win.registry.ssl3.client.enabled"
+    key_path: HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Control\\SecurityProviders\\SCHANNEL\\Protocols\\SSL\\3.0\\Client
```

Comment:
there is actually a space between SSL and 3.0 in the registry key path

Reply:
It was not working without escaping ... I'll accept the PR and re-check the code...